go.mod

```diff
@@ -17,9 +18,9 @@ require (
 	github.com/nxadm/tail v1.4.4
 	github.com/onsi/ginkgo v1.14.0
 	github.com/onsi/gomega v1.10.1
-	github.com/planetdecred/dcrlibwallet v1.6.2-0.20220203081357-951ce632e9d2
+	github.com/planetdecred/dcrlibwallet v1.6.2-0.20220315164532-e871b673497a // indirect
 	github.com/yeqown/go-qrcode v1.5.1
 	golang.org/x/exp v0.0.0-20210722180016-6781d3edade3
 	golang.org/x/image v0.0.0-20210628002857-a66eb6448b8d
 	golang.org/x/text v0.3.7
-)
+)
```

Comment:
Need to run go mod tidy. That will restore the newline at the end of this file and also fix the indirect imports in the file.